specification/containerservice/resource-manager/readme.go.md

```diff
@@ -67,6 +68,16 @@ batch:
   - tag: package-2017-07
 ```
 
+### Tag: package-preview-2022-09 and go
```

Comment:
We do not recommend to update the configuration here anymore because track 1 GO SDK has been deprecated.
And the track 1 validation pipeline has been turned off, therefore these things will never be validated.

Reply:
Get it, removed.